apps/rich-text-versioning/src/locations/Dialog.tsx

```diff
@@ -57,10 +59,42 @@ const Dialog = () => {
     return ids;
   };
 
+  const getAssets = async (assetIds: string[]): Promise<AssetProps[]> => {
+    if (assetIds.length === 0) return [];
+
+    try {
+      const fetchedAssets = await sdk.cma.asset.getMany({
+        query: {
+          select: 'sys.id,fields.title',
+          'sys.id[in]': assetIds.join(','),
+        },
+      });
+      return fetchedAssets.items;
+    } catch (error) {
+      console.error('Error fetching assets:', error);
+      return [];
+    }
+  };
+
+  const getEntries = async (entryIds: string[]): Promise<EntryProps[]> => {
+    if (entryIds.length === 0) return [];
+
+    try {
+      const fetchedEntries = await sdk.cma.entry.getMany({
+        query: {
+          'sys.id[in]': entryIds.join(','),
+        },
+      });
```

Comment:
💭 Could we also select the fields we need from the entries, instead of the whole entry? Does it make sense to do it?